types.ts

```diff
@@ -970,6 +977,45 @@ export type VariousSupportedAuthentication =
  */
 export type VariousSupportedAuthenticationTypes = $Values<Pick<VariousSupportedAuthentication, 'type'>>;
 
+/**
+ * @hidden
+ */
+export enum ReservedAuthenticationKeys {
+  /**
+   * References the default user authentication of the pack.
+   */
+  Default = 'DEFAULT',
```

Comment:
Nit, I think we have a standard enum style in the SDK, can't remember if it's TitleCase or camelCase, but should use it here. If you're trying to use keys that are unlikely to collide with user keys, would go even more distinctive.

Reply:
now: 
```
export enum ReservedAuthenticationNames {
  Default = 'defaultAuthentication',
  System = 'systemAuthentication',
}
```